src/posthog-surveys.ts

```diff
@@ -1,6 +1,5 @@
 import { SURVEYS } from './constants'
 import { SurveyManager } from './extensions/surveys'
-import { getSurveySeenStorageKeys } from './extensions/surveys/surveys-extension-utils'
```

Comment:
you're right @daibhin this should be lintable...

Reply:
yeah something like not allowing importing from ./extensions/* from main posthog-* files